Lib/test/support/strace_helper.py

```diff
@@ -91,7 +91,10 @@ def _make_error(reason, details):
         res, cmd_line = run_python_until_end(
             "-c",
             textwrap.dedent(code),
-            __run_using_command=[_strace_binary] + strace_flags)
+            __run_using_command=[_strace_binary] + strace_flags,
+            # Don't want to trace our JIT's own mmap and mprotect calls:
+            PYTHON_JIT="0",
```

Comment:
Guessing this showed up in the `test_fileio` test `test_syscalls_read`?

I have keeping an eye on / working through a couple other issues where mmap(NULL) and mprotect show up, trying to figure out if just excluding those two from that test would be good/effective (memory allocation happening in reading files is fine / not what the test is focused on).

Reply:
Yeah, I was wondering if this was susceptible to catching badly-timed arena allocations and such too. Might make sense to have an option to ignore the `mmap`/`munmap`/`mprotect` family of functions instead of this ad-hoc fix.

I'm just not familiar with how this is being used, and didn't want to change the meaning/behavior of the tests too dramatically.